src/components/views/rooms/EntityTile.js

```diff
@@ -29,6 +29,23 @@ var PRESENCE_CLASS = {
     "unavailable": "mx_EntityTile_unavailable"
 };
 
+
+function presenceClassForMember(presence_state, last_active_ago) {
```

Comment:
variables are still wrong case, but whatever. let's just try to avoid adding to the mess in future.....